src/explore-education-statistics-admin/src/components/PageHeader.tsx

```diff
@@ -71,27 +82,29 @@ const PageHeader = ({ wide }: Props) => {
               Explore education statistics
             </Link>
 
-            <button
-              type="button"
-              className={classNames(
-                'govuk-header__menu-button',
-                'govuk-js-header-toggle',
-                { 'govuk-header__menu-button--open': menuOpen },
-              )}
-              aria-controls="navigation"
-              aria-expanded={menuOpen}
-              aria-label="Show or hide Top Level Navigation"
-              onClick={() => setMenuOpen(!menuOpen)}
+            <nav
+              aria-label="Top Level Navigation"
+              className="govuk-header__navigation "
             >
-              Menu
-            </button>
-            <nav>
+              <button
+                type="button"
+                className={classNames(
+                  'govuk-header__menu-button',
```

Comment:
Just wondering if we can drop these classes into one string to be a bit more succinct? i.e. 
`'govuk-header__menu-button govuk-js-header-toggle'`